### ISSUE TITLE
Disallow credentials with VC/JSON and VC/JWT members that do not encode the same value

### ISSUE THREAD

**Issue by mirceanis** [decentralized-identity/did-jwt-vc#64]
As discussed in [PR#63](https://github.com/decentralized-identity/did-jwt-vc/pull/63/files#r583764430) there is potential ambiguity during VC/JWT <-> VC/JSON transformation if both entries are present but encode different values.

During VC or VP creation these methods should throw an error in that case and the transformations that are performed for convenience could override the entries that don't correspond to the respective encoding.

It is very inefficient to keep both representations in a credential payload, but that is an ambiguity with the spec we're working on so it must be handled.
Let's discuss the best approach before venturing a fix.

**Comment by mirceanis** [comment]
Thank you for the summary.
Indeed it's not ok to drop information. Fixing this particular GH issue means throwing an error when information would be overridden or dropped, thus not allowing any ambiguity.

**Comment by OR13** [comment]
@mirceanis yes, we have a similar issue we are tracking in our implementation.
